lua-tree/share/lua/5.1/kong/db/schema/init.lua

```diff
@@ -1243,7 +1249,7 @@ local function run_entity_check(self, name, input, arg, full_check, errors)
 
   else
     local error_fmt = validation_errors[name:upper()]
-    local err = error_fmt and error_fmt:format(err) or err
+    err = error_fmt and error_fmt:format(err) or err
```

Comment:
@mikefero it seems regenerating the lua-tree is causing [this](https://github.com/Kong/goks/runs/6003993911?check_suite_focus=true) failure

If I add the `local` definition back, tests pass. Shall we include that in the patch as well? (I have absolutely no idea why that is happening)

Reply:
Yeah the `local` is needed and should have been part of the patch. If I remember correctly this is related to [gopher-lua issue](https://github.com/yuin/gopher-lua/issues/355) we discovered awhile back.

Was this change missing from the patch or does the patch need to be updated again?